crates/bevy_ui/src/node_bundles.rs

```diff
@@ -124,6 +124,27 @@ pub struct TextBundle {
     pub computed_visibility: ComputedVisibility,
     /// Indicates the depth at which the node should appear in the UI
     pub z_index: ZIndex,
+    /// The background color that will fill the containing node
+    pub background_color: BackgroundColor,
+}
+
+impl Default for TextBundle {
+    fn default() -> Self {
+        Self {
+            text: Default::default(),
+            calculated_size: Default::default(),
+            // Transparent background
+            background_color: Color::NONE.into(),
```

Comment:
```suggestion
            background_color: BackgroundColor(Color::NONE),
```

One less function call?